enterprise/frontend/src/embedding-sdk/components/public/CreateQuestion/CreateQuestion.tsx

```diff
@@ -1,23 +1,115 @@
-import { QuestionEditor } from "embedding-sdk/components/private/QuestionEditor";
+import { useState } from "react";
 
-import type { InteractiveQuestionProps } from "../InteractiveQuestion";
+import { FlexibleSizeComponent } from "embedding-sdk";
+import { useInteractiveQuestionContext } from "embedding-sdk/components/private/InteractiveQuestion/context";
+import { SaveQuestionModal } from "metabase/containers/SaveQuestionModal";
+import { Button, Flex } from "metabase/ui";
 
-type CreateQuestionProps = Omit<InteractiveQuestionProps, "questionId">;
+import {
+  InteractiveQuestion,
+  type InteractiveQuestionProps,
+} from "../InteractiveQuestion";
+
+type CreateQuestionProps = Partial<
+  Omit<InteractiveQuestionProps, "questionId" | "children">
+>;
 
 export const CreateQuestion = ({
-  plugins,
   onSave,
-  onBeforeSave,
-  entityTypeFilter,
-  isSaveEnabled,
-  saveToCollectionId,
-}: CreateQuestionProps = {}) => (
-  <QuestionEditor
-    plugins={plugins}
-    onBeforeSave={onBeforeSave}
-    onSave={onSave}
-    entityTypeFilter={entityTypeFilter}
-    isSaveEnabled={isSaveEnabled}
-    saveToCollectionId={saveToCollectionId}
-  />
-);
+  isSaveEnabled = true,
+  ...props
+}: CreateQuestionProps = {}) => {
+  const [isSaveModalOpen, setSaveModalOpen] = useState(false);
+
+  return (
+    <InteractiveQuestion
+      {...props}
+      isSaveEnabled={isSaveEnabled}
+      onSave={(question, context) => {
+        if (question) {
+          setSaveModalOpen(false);
+          onSave?.(question, context);
+        }
+      }}
+    >
+      <CreateQuestionDefaultView
+        isSaveModalOpen={isSaveModalOpen}
+        setSaveModalOpen={setSaveModalOpen}
+      />
+    </InteractiveQuestion>
+  );
+};
+
+export const CreateQuestionDefaultView = ({
```

Comment:
Oisin is cooking a [better default layout](https://www.figma.com/proto/Vkrs53Rit3EPmCaIkI0ZBf/SDK-Default-Layout?page-id=96%3A9544&node-id=96-9565&node-type=frame&t=RTgs7uSw5DfUuUK3-0&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=96%3A11359&show-proto-sidebar=1) right now. I'll likely make a follow-up PR to reuse that layout here.